scripts/scil_compute_tracking_dipy.py

```diff
@@ -0,0 +1,265 @@
+#! /usr/bin/env python
```

Comment:
While we are here, you could rename this to simply `scil_compute_local_tracking.py`. I don't think we should port the other local tracking script.

Reply:
The other script has "_dipy" at the end, do we care to put it on both, remove one ?

Reply:
We should probably remove it in for the other as well.

However, this would need to be coordinated with @GuillaumeTh and @arnaudbore for Tractoflow.